Sources/POSIX/Spawn.swift

```diff
@@ -17,11 +17,13 @@ extension SpawnError : CustomStringConvertible {
     }
 }
 
+#if !os(watchOS) && !os(tvOS)
```

Comment:
Instead of using #if, you should use @available annotation which is appropriate to accomplish your intentions.